src/GameServer/Bot/AI/BotSupportPlanner.js

```
@@ -253,6 +253,104 @@ function refreshThresholdMs(skill) {
+
+    // Direct player-requested buffs may pass only the player as `members`,
+    // while the native party aura still reaches the complete party.
+    const providerSession = provider?.session;
+    const leaderSession = providerSession?.partyCompanion === true && providerSession.followPlayerSession
+        ? providerSession.followPlayerSession
+        : providerSession;
+    if (leaderSession) {
+        try {
+            const PartyAwareness = invoke('GameServer/Bot/AI/PartyAwareness');
+            PartyAwareness.partyActors(leaderSession).forEach(addActor);
+        } catch (_) {}
```

ESLint

[error] Empty block statement.

(no-empty)